src/js/edu-benefits/components/benefits-eligibility/BenefitsSelectionFields.jsx

```diff
@@ -49,39 +50,36 @@ export default class BenefitsSelectionFields extends React.Component {
 
     return (<fieldset>
       <legend className="hide-for-small-only">Benefits eligibility</legend>
-      <div className="input-section" style={{ display: 'none' }}>
-        <ErrorableRadioButtons
-            label="Are you applying using your own benefits or those of a spouse or parent?"
-            name="applyingUsingOwnBenefits"
-            options={ownBenefitsOptions}
-            value={this.props.data.applyingUsingOwnBenefits}
-            onValueChange={(update) => {this.props.onStateChange('applyingUsingOwnBenefits', update);}}/>
-      </div>
-      <p>Select the benefit that is the best match for you. If you are eligible for more than one, we can help you make the best choice.</p>
       <div className="input-section">
-        <ExpandingGroup open={this.props.data.chapter33} additionalClass="edu-benefits-chapter33-group">
+        <ErrorableGroup
+            label="Select the benefit that is the best match for you. If you are eligible for more than one, we can help you make the best choice."
+            validation={this.props.data.chapter33 || this.props.data.chapter30 || this.props.data.chapter1606 || this.props.data.chapter32}
+            errorMessage="Please select at least one benefit"
+            isDirty={this.props.data.checkedBenefit.dirty}>
```

Comment:
I'm just using it as a way to trigger the error state for the group, since we don't have dirty states for checkboxes. The value for it is never set, but dirty is set to true along with other fields on the page when you try to move to the next page.